tmt/utils.py

```diff
@@ -276,41 +347,74 @@ def _log(self, message):
                 log.write(datetime.datetime.utcnow().strftime('%H:%M:%S') + ' '
                           + remove_color(message) + '\n')
 
-    def print(self, key, value=None, color=None, shift=0, err=False):
+    def print(
```

Comment:
Dropping `level` is certainly possible, however, it'd be 3rd type of logging function, besides `warn` and `fail` (so-called `SemanticLoggerFnType`) and `verbose` + `debug` (`BaseLoggerFnType`). I probably decided to rather add the parameter instead of the third logging function type, but since `level` is unused and you confirmed such behavior is expected, I suppose the third logging type it is.

Reply:
Added third type in 3a48196f2d8f4c196850c807abcc49f325198e6f.

I believe this part of the codebase will get refactored in the future, possibly reducing the number of types involved :)